src/models/DefectsDto.js

```diff
@@ -0,0 +1,61 @@
+const AWS = require('aws-sdk')
+const config = require('../config/config')
+const dbClient = new AWS.DynamoDB.DocumentClient(
+  (config.ENV === 'local') ? { region: config.OFFLINE.DYNAMODB_REGION, endpoint: config.OFFLINE.DYNAMODB_ENDPOINT } : {})
+
+class DefectsDto {
```

Comment:
Seeing the content of DefectsDto it seems more of a DAO(Data Access Object) and not DTO, so may be DefectsDAO?